model/src/main/proto/topic.proto

```diff
@@ -203,6 +203,9 @@ message PromotedStory {
 
   // The thumbnail that should be displayed for this promoted story.
   LessonThumbnail lesson_thumbnail = 9;
+
+  // The indicator to handle if story is completed.
+  string completed_story_topic_id = 10;
```

Comment:
Why do we need the topic ID here? I thought what we needed was an indicator whether the topic corresponding to this story can be considered 'complete' (ie, at least one story is completed). Couldn't this be a boolean? E.g.: ``is_topic_learned``.

Reply:
I was thinking if Id would be more informative, if one or more topics were completed.

Reply:
What exactly is this ID used for in the UI? That part isn't clear to me, and it seems like the specific need for having this be an ID versus a boolean.

Reply:
I used Id to check if any ongoing topic Id matches with completed story's topic Id.  But as per the [comment](https://github.com/oppia/oppia-android/pull/2272#discussion_r560729133) I could change the implementation and make use of boolean as indicator to check if topic was learned.